gcloud/datastore/test___init__.py

```diff
@@ -30,59 +30,86 @@ def _callFUT(self, dataset_id=None):
         from gcloud.datastore import set_default_dataset_id
         return set_default_dataset_id(dataset_id=dataset_id)
 
-    def _monkey(self, implicit_dataset_id, httplib=None, app_identity=None):
-        from contextlib import nested
+    def _monkeyEnviron(self, implicit_dataset_id):
         import os
-
         from gcloud._testing import _Monkey
         from gcloud.datastore import _DATASET_ENV_VAR_NAME
+        environ = {_DATASET_ENV_VAR_NAME: implicit_dataset_id}
+        return _Monkey(os, getenv=environ.get)
+
+    def _monkeyImplicit(self, connection=None, app_identity=None):
```

Comment:
Worse than deprecated, it was removed (at least by 3.4), and without ever triggering any deprecation warnings that I know about.